aiosmtpd/smtp.py

```diff
@@ -723,10 +737,26 @@ async def smtp_AUTH(self, arg: str) -> None:
             if status is not None:  # pragma: no branch
                 await self.push(status)
 
-    async def _auth_interact(self, server_message) -> _TriStateType:
+    async def _auth_interact(
+            self,
+            server_message,
+            log_client_response: bool = False
+    ) -> _TriStateType:
+        """
+        Sends a challenge, and wait for client response.
+
+        :param server_message: Challenge to send
+        :param log_client_response: Perform logging of client's response.
+            WARNING: Might cause leak of sensitive information! Do not turn on
+            unless _absolutely_ necessary!
+        """
```

Comment:
Come to think of it, logging AUTH interaction _might_ be useful in some situations.

So rather than hardcoding no logging, we'll provide a default-disabled logging here.